src/screens/FeedScreen/RenderList.js

```diff
@@ -256,8 +176,7 @@ const RenderListFeed = (props) => {
             statusVote={voteStatus}
             showScoreButton={showScoreButton}
             onPressScore={() => showScoreAlertDialog(item)}
-            isSelf={isSelf}
-            isShowDM
+            isSelf={item.anonimity ? false : selfUserId === item.actor.id}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The prop `isSelf` has been replaced with a conditional check `item.anonimity ? false : selfUserId === item.actor.id`. Ensure that `selfUserId` is defined and correctly set elsewhere in the component.



<!-- This is an auto-generated comment by OSS CodeRabbit -->